src/diff/index.js

```diff
@@ -75,13 +75,14 @@ export function diff(
 	if (newVNode.constructor != UNDEFINED) return NULL;
 
 	// If the previous diff bailed out, resume creating/hydrating.
-	if (oldVNode._flags & MODE_SUSPENDED) {
-		isHydrating = !!(oldVNode._flags & MODE_HYDRATE);
-		if (oldVNode._component._excess) {
```

Comment:
I have yet to dig into an attempt to reproduce but the removal of this check is causing CI failures in iso. Was it intentionally removed? I'd think we'd still need it.